Userland/Services/WebDriver/Session.h

```diff
@@ -28,14 +29,26 @@ class Session {
 
     unsigned session_id() const { return m_id; }
 
+    struct Window {
+        String handle;
+        RefPtr<WebContentConnection> web_content_connection;
+    };
+
     WebContentConnection& web_content_connection()
     {
-        VERIFY(m_web_content_connection);
-        return *m_web_content_connection;
+        return *(m_windows.get(m_current_window_handle).value()->web_content_connection);
```

Comment:
There's a lot going on in this line :sweat_smile:  let's break it up into a couple lines